src/main/resources/view/NameCard.fxml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.ColumnConstraints?>
+<?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.layout.HBox?>
+
+<GridPane id="nameCard" fx:id="nameCard" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1"
+          HBox.hgrow="ALWAYS">
+    <columnConstraints>
+        <ColumnConstraints hgrow="SOMETIMES" minWidth="10"/>
+    </columnConstraints>
+    <HBox alignment="CENTER_LEFT" minHeight="-Infinity" GridPane.columnIndex="0" >
+        <padding>
+            <Insets top="5" right="5" bottom="5" left="15"/>
+        </padding>
+        <Label fx:id="favourite" text="\$favourite" />
```

Comment:
That was the original idea, but it would be a poor design choice because for a person with a really long name like Wilhelmina Alexandrine Helena Louise Maria Frederica von Brandenburg-Schwedt, the icon would appear on the right side of the name which means that it would be hidden from the user on first glance, and the user needs to scroll all the way to the right to see the star icon. Hence to work around this caveat, it is place on the left which offsets the name and gives the list a rather unique styling choice of indentation! Two birds in one stone.

Reply:
> That was the original idea, but it would be a poor design choice because for a person with a really long name like Wilhelmina Alexandrine Helena Louise Maria Frederica von Brandenburg-Schwedt, the icon would appear on the right side of the name which means that it would be hidden from the user on first glance, and the user needs to scroll all the way to the right to see the star icon. Hence to work around this caveat, it is place on the left which offsets the name and gives the list a rather unique styling choice of indentation! Two birds in one stone.

@Pluiexo Is it possible to wrap the text instead? like
```
Wilhelmina Alexandrine Helena
Louise Maria Frederica von
Brandenburg-Schwedt
```

But i understand where you're coming from. We can keep it as is for now, if there's time, perhaps we can retinker this portion. 👍

Reply:
It is possible to wrap text, but it comes with the caveat of the items in the left side being of inconsistent height, which could make the UI list view look really out of shape. At this point it really is pick your own poison considering the limitations of JavaFx library 🥲 .